cli/command/network/create.go

```diff
@@ -61,7 +69,7 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.VarP(&options.driverOpts, "opt", "o", "Set driver specific options")
 	flags.Var(&options.labels, "label", "Set metadata on a network")
 	flags.BoolVar(&options.internal, "internal", false, "Restrict external access to the network")
-	flags.BoolVar(&options.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(options.ipv6, "ipv6", false, "Enable IPv6 networking")
```

Comment:
LOL perhaps we need to create our own "optional bool" so that we could have `nil` as default. 🤔 

Perhaps the flag-description should be updated though; something like "auto" (or "determined by daemon"?);

```suggestion
	flags.BoolVar(options.ipv6, "ipv6", false, "Enable or disable IPv6 networking (default is auto)")
```

or should we consider the default to be "true"?

```suggestion
	flags.BoolVar(options.ipv6, "ipv6", true, "Enable or disable IPv6 networking (default is enabled, if supported)")
```

Reply:
> or should we consider the default to be "true"?

I think we shouldn't do that, at least for now. One reason is the discussion we had regarding ip6tables. Another reason, IPv6 isn't supported by all drivers currently (eg. overlay).

If we want to enable IPv6 by default, I think it'd need to be done at the API level, in a new API version.